hack/examples/bases/nutanix/cluster/kustomization.yaml.tmpl

```diff
@@ -5,7 +5,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- https://raw.githubusercontent.com/nutanix-cloud-native/cluster-api-provider-nutanix/main/templates/cluster-template-topology.yaml
+- https://raw.githubusercontent.com/nutanix-cloud-native/cluster-api-provider-nutanix/51aa55fe8b194a261bc3cc4e6d24f5d430ec055a/templates/cluster-template-topology.yaml
```

Comment:
we should take main capx branch now

Reply:
https://raw.githubusercontent.com/nutanix-cloud-native/cluster-api-provider-nutanix/main/templates/cluster-template-topology.yaml

Reply:
@deepakm-ntnx thats what we were doing before and it broke all PRs because of an upstream change
I would like to avoid that in the future until there is a tag that can be used.